rt/rs/microprofile-client/src/main/java/org/apache/cxf/microprofile/client/MicroProfileClientProviderFactory.java

```diff
@@ -91,16 +91,11 @@ protected void setProviders(boolean custom, boolean busGlobal, Object... provide
 
     public List<ResponseExceptionMapper<?>> createResponseExceptionMapper(Message m, Class<?> paramType) {
 
-        List<ResponseExceptionMapper<?>> candidates = new LinkedList<>();
-
-        for (ProviderInfo<ResponseExceptionMapper<?>> em : responseExceptionMappers) {
-            if (handleMapper(em, paramType, m, ResponseExceptionMapper.class, true)) {
-                candidates.add(em.getProvider());
-            }
-        }
-        if (candidates.size() == 0) {
+        if (responseExceptionMappers.isEmpty()) {
             return Collections.emptyList();
         }
+        List<ResponseExceptionMapper<?>> candidates =
```

Comment:
Even shorter: 
```
return Collections.unmodifiableList(responseExceptionMappers
    .stream()
    .map(ProviderInfo::getProvider)
    .sorted(new ResponseExceptionMapperComparator())
    .collect(Collectors.toList()));
```

Reply:
@reta - thanks for the suggestion - I'm making that change and squashing now.